client/packages/programs/src/JsonForms/components/Search/usePatientSearchQuery.tsx

```diff
@@ -1,8 +1,8 @@
-import { Utils } from '@openmsupply-client/system/src/Patient/api/hooks/utils';
 import { PatientSchema } from '@openmsupply-client/programs';
+import { usePatient } from '@openmsupply-client/system';
 
 export const usePatientSearchQuery = () => {
-  const { data, error, isLoading, mutateAsync } = Utils.usePatientSearch();
+  const { data, error, isLoading, mutateAsync } = usePatient.utils.search();
```

Comment:
Its IMHO still a bit messy but I tried to unify which search function is actually used...